src/test/unit/services/sample/hmc_nuts_dense_e_adapt_parallel_match_test.cpp

```diff
@@ -97,26 +114,38 @@ TEST_F(ServicesSampleHmcNutsDenseEAdaptParMatch, single_multi_match) {
   }
 
   std::vector<Eigen::MatrixXd> par_res;
+  std::vector<std::string> par_metrics;
   for (int i = 0; i < num_chains; ++i) {
     auto par_str = par_parameters[i].get_stream().str();
     auto sub_par_str = par_str.substr(par_str.find("Elements") - 1);
     std::istringstream sub_par_stream(sub_par_str);
     Eigen::MatrixXd par_mat
         = stan::test::read_stan_sample_csv(sub_par_stream, 80, 9);
     par_res.push_back(par_mat);
+
+    par_metrics.push_back(ss_metric[i].str());
+    rapidjson::Document document;
+    ASSERT_FALSE(document.Parse<0>(par_metrics[i].c_str()).HasParseError());
```

Comment:
[Optional] it might be nice to pull this "assert that the JSON in this string parses" logic into a test utility function/macro so we can re-use it rather than needing to remember the RapidJSON incantation.